### PR TITLE
Use a premailer to inline all CSS

### PR DESCRIPTION
**Pull request by AeroCross**
This will make emails render correctly in clients like Gmail.

### Before

screen shot 2016-06-20 at 11 01 04 pm

### After

screen shot 2016-06-20 at 10 56 04 pm

WOOOO

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -49,6 +49,10 @@ gem 'rails_12factor'
 
 gem "virtus"
 
+# For proper email rendering in Gmail
+gem 'nokogiri'
```
Comment (AeroCross) [inline Gemfile]: Nokogiri is a [dependency of `premailer-rails`](https://github.com/fphilipe/premailer-rails#installation)

Comment (ejoubaud) [inline Gemfile]: Funny that it doesn't include it itself 😕

**Comment by ejoubaud** [comment]
Beautiful 😻 🚢

**Pull request merged**